pkgs/development/python-modules/stack-data/default.nix

```
@@ -1,55 +1,43 @@
-{ asttokens
-, buildPythonPackage
-, cython
-, executing
+{ buildPythonPackage
 , fetchFromGitHub
-, git
 , lib
-, littleutils
-, pure-eval
-, pygments
+, python3
+, pythonPackages
 , pytestCheckHook
-, setuptools-scm
-, toml
-, typeguard
 }:
 
 buildPythonPackage rec {
   pname = "stack-data";
-  version = "0.2.0";
-  format = "setuptools";
+  version = "0.6.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "alexmojaki";
     repo = "stack_data";
     rev = "v${version}";
```

Avoids branch/tags confusion. Helps with batch updates.

```suggestion
    rev = "refs/tags/v${version}";
```